src/resources/views/costs.blade.php

```diff
@@ -21,7 +21,7 @@
                     <dt>Destination</dt>
                     <dd>{{ $route->destination_location()->name }}</dd>
                     <dt>Info</dt>
-                    <dd>{{ $route->info_text ?? "Your administrator can put a text about how to create the contract here." }}</dd>
+                    <dd>{!! $route->info_text ?? "Your administrator can put a text about how to create the contract here." !!}</dd>
```

Comment:
Note that I unescape the content to render the supplied HTML. This was a "quick fix" for my personal installation and probably need to be more robust to merge.

Reply:
Yes, I will not merge unescaped html. I'm open for integrating seat-info or the editor e.g. squad descriptions use.